src/rulelib.cpp

```diff
@@ -917,74 +870,131 @@ ruleset_print(ruleset_t *rs, rule_t *rules, int detail)
 }
 
 void
-ruleset_entry_print(ruleset_entry_t *re, int n, int detail)
+ruleset_entry_print(ruleset_entry_t *re, int nsamples, int detail)
 {
 	printf("%d captured; \n", re->ncaptured);
 	if (detail)
-		rule_vector_print(re->captures, n);
+		rule_vector_print(re->captures, nsamples);
 }
 
 void
-rule_print(rule_t *rules, int ndx, int n, int detail)
+rule_print(rule_t *rules, int ndx, int nsamples, int detail)
 {
 	rule_t *r;
 
 	r = rules + ndx;
-	printf("RULE %d: ( %s ), support=%d, card=%d:",
+	printf("RULE %d: ( %s ), support=%d, card=%d",
 	    ndx, r->features, r->support, r->cardinality);
-	if (detail)
-		rule_vector_print(r->truthtable, n);
+	if (detail) {
```

Comment:
Indentation looks strange here

Reply:
Thanks for pointing that out, I must have edited it on Windows or something.